contrib_bots/lib/CommuteBot/doc.md

```diff
@@ -0,0 +1,57 @@
+This bot will allow briefings of estimated travel times, distances and fare information for transit travel.
+
+It can respond to: departure times, arrival times, user preferences (toll avoidance, highway avoidance) and a mode of transport
+
+It can output: fare information, more detailed addresses on origin and destination, duration in traffic information, metric and imperical units and information in various languages.
+
+The bot will respond to the same stream input was in. And if called as private message, the bot will reply with a private message.
+
+To setup the bot, you will first need to move settings.ini into the user home directory and add an API key.
+
+Move <pre><code>~/zulip/contrib_bots/lib/CommuteBot/settings.ini</code></pre> into <pre><code>~/settings.ini</code></pre>
+
+To add an API key, please visit: https://developers.google.com/maps/documentation/distance-matrix/start to retrieve a key and copy your api key into settings.ini
+
+Sample input and output:
+
+<pre><code>@commute help</code></pre>
+
+<pre><code>Obligatory Inputs:
+    Origin e.g. origins=New+York,NY,USA
+    Destination e.g. destinations=Chicago,IL,USA
+
+Optional Inputs:
+    Mode Of Transport (inputs: driving, walking, bicycling, transit)
+    Default mode (no mode input) is driving
+    e.g. mode=driving or mode=transit
+    Units (metric or imperial)
+    e.g. units=metric
+    Restrictions (inputs: tolls, highways, ferries, indoor)
+    e.g. avoid=tolls
+    Departure Time (inputs: now or (YYYY, MM, DD, HH, MM, SS) departing)
+    e.g. departure_time=now or departure_time=2016,12,17,23,40,40
+    Arrival Time (inputs: (YYYY, MM, DD, HH, MM, SS) arriving)
+    e.g. arrival_time=2016,12,17,23,40,40
+    Languages:
+    Languages list: https://developers.google.com/maps/faq#languagesupport)
+    e.g. language=fr
```

Comment:
The language option is changing the language for all subsequent calls.

![screen shot 2016-12-19 at 4 57 29 pm](https://cloud.githubusercontent.com/assets/3491502/21319134/47338f36-c60c-11e6-88f2-b7e75a1a397d.png)

I'm not sure what's the better option here - reverting to English or updating the docs, so that the user knows what's happening. 

Other issue with the language option is that it doesn't change the language of the text you've typed in your code - I'm not sure language support should be the focus of this bot for now, but this is something to think about if growing this feature.